js/background.js

```diff
@@ -230,6 +230,9 @@
     window.feeds = [];
     window.lokiMessageAPI = new window.LokiMessageAPI(ourKey);
     window.lokiPublicChatAPI = new window.LokiPublicChatAPI(ourKey);
+    // set up mixpanel
+    window.mixpanel = window.mixpanel || new window.LokiMixpanelAPI(); // incase account manager already set it up
```

Comment:
remove if you already set it up in `preload.js` ;)

Reply:
removed.